src/Runner/Runner.php

```diff
@@ -134,8 +134,16 @@ public function fix()
             Tokens::clearCache();
 
             if ($fixInfo) {
-                $name = $this->directory->getRelativePathTo($file);
-                $changed[$name] = $fixInfo;
+                $path = $file->getPathname();
+
+                /* workaround for Symfony/Finder setting doubled slash for root folders in
+                SplFileInfo. See `RecursiveDirectoryIterator::current()`. */
+                if (substr($path, 0, 2) === '//') {
```

Comment:
The example above is fixing the case for double `//` at the start of a path which is caused by finder's bug. Doubled `DIRECTORY_SEPARATOR` bug is caused by `$pathsByType['dir'][] = $path.DIRECTORY_SEPARATOR;` fixed by ` $pathsByType['dir'][] = ($path === '/' || substr($path, -1, 1) !== DIRECTORY_SEPARATOR) ? $path : substr($path, 0, strlen($path) - 1);`